src/docs/Chakra.mdx

```diff
@@ -150,13 +150,13 @@ in the `/src/index.ts` file. Note that not _all_ Chakra components are re-export
 because we want to limit what developers should use in consuming applications.
 They include:
 
-- `Box`, `Center`, `Circle`, `Grid`, `GridItem`, `HStack`, `Square`, `Stack`, `VStack`
+`Box`, `Center`, `Circle`, `Grid`, `GridItem`, `HStack`, `Square`, `Stack`, `VStack`
 
-Note: These components are harder to have snapshot tests for because, internally,
+**Note**: These components are harder to have snapshot tests for because, internally,
```

Comment:
Let's make sure to create a ticket to start replacing these notes and warnings with the `Banner` component.